api/envoy/admin/v3/config_dump.proto

```diff
@@ -58,9 +81,13 @@ message UpdateFailureState {
   // Details about the last failed update attempt.
   string details = 3;
 
-  // This is the version of the rejected resource.
+  // This is the version of the latest update for this resource.
   // [#not-implemented-hide:]
   string version_info = 4;
+
+  // Error status code of this config.
+  // [#not-implemented-hide:]
+  ConfigErrorStatus status = 5;
```

Comment:
Does this belong in `UpdateFailureState`, or should it go directly in the places where `UpdateFailureState` is used, as a separate field?  The reason I ask is that the `REQUESTED` enum value is not actually a failure, so it seems inconsistent to reflect that state in this field.

Also, if we pull it out, we can add an `ACKED` value to the enum to reflect that case as well.

Reply:
Promoted this field to the same level of `error_state`.